homeassistant/util/ssl.py

```diff
@@ -127,6 +134,19 @@ def get_default_no_verify_context() -> ssl.SSLContext:
     return _DEFAULT_NO_VERIFY_SSL_CONTEXT
 
 
+def server_context_no_verify(
+    ssl_cipher_list: SSLCipherList = SSLCipherList.PYTHON_DEFAULT,
+) -> ssl.SSLContext:
+    """Return a SSL context which a specific ssl cipher."""
+    if ssl_cipher_list == SSLCipherList.INTERMEDIATE:
+        return _INTERMEDIATE_NO_VERIFY_SSL_CONTEXT
+    if ssl_cipher_list == SSLCipherList.MODERN:
+        return _MODERN_NO_VERIFY_SSL_CONTEXT
+    if ssl_cipher_list == SSLCipherList.INSECURE:
+        return _INSECURE_NO_VERIFY_SSL_CONTEXT
+    return _DEFAULT_NO_VERIFY_SSL_CONTEXT
```

Comment:
This can be written as a dict `.get()` lookup with `_DEFAULT_NO_VERIFY_SSL_CONTEXT` being the default

Reply:
Done